SCons/Tool/msvc.xml

```diff
@@ -212,6 +212,18 @@ Example:
 <example_commands>
 env['PCH'] = 'StdAfx.pch'
 </example_commands>
+
+<para>
```

Comment:
Don't think you need this. The expectation is that all variables should support such.

Reply:
your saying the docs? Okay I'll revert it